source/_drafts/v3-planning.md

```diff
@@ -0,0 +1,21 @@
+title: Marionette V3 -- The Plan
+tags:
+  - plans
+  - v3
+---
+
+This past friday the Marionette Team got together and did some v3 planning for the next release of marionette.
+The focus on of our conversation was on looking into adding the missing application layer pieces that are very much needed within Marionette.
+Over the past few years there has been a community effort to establish patterns to allow people to build large applications in Marionette.
+We are exploring ways that we can codify the concepts into the core.
+
+[@jmeas](https://github.com/jmeas/) has done some great work in this area via the creation of a library called [base-router](https://github.com/jmeas/backbone.base-router). We want to take this work and build ontop of it to create a router that has some new concepts such as state based routing, and named routes. There will most likely be more features, inspired both from the great ember router and angular ui router.
```

Comment:
Just got around to reading this

> inspired both from the great ember router and angular ui router.

inspired by the great router implementations of Ember and angular-ui.

Probably should also link their docs